locale/fi/LC_MESSAGES/django.po

```diff
@@ -580,6 +576,10 @@ msgstr ""
 "          Palauta ryhmässä: %(collaborators)s\n"
 "          "
 
+#: course/templates/course/_siblings.html:4
+msgid "Pagination"
+msgstr "Sivuilla siirtyminen"
```

Comment:
This probably should be iterated at some point, but I guess it's fine. A bit problematic that we add translation for a single word "Pagination", which _could_ be used somewhere else too. For now it is not, thus fine.

Reply:
Could the context be provided?

Reply:
@etanttila do you mean add it as a comment in the locale file?

Reply:
Technically it could be possible to use actual id as the reference and not English translation, but that requires that there would be another file with the English string too.

Reply:
@mtthwhggns I meant the PO msctxt context, but wasn't very familiar with it. But if it requires that the English string appears somewhere else as well like @raphendyr says, I guess it's "sen ajan murhe".